Makefile

```diff
@@ -149,9 +149,9 @@ test-e2e: ##@tests Run e2e tests
 	build/env.sh go test -timeout 5m ./e2e/node/...
 	build/env.sh go test -timeout 15m ./e2e/jail/...
 	build/env.sh go test -timeout 20m ./e2e/rpc/...
-	build/env.sh go test -timeout 10m ./e2e/whisper/...
+	build/env.sh go test -timeout 20m ./e2e/whisper/...
 	build/env.sh go test -timeout 10m ./e2e/transactions/...
-	build/env.sh go test -timeout 10m ./cmd/statusd
+	build/env.sh go test -timeout 20m ./cmd/statusd
```

Comment:
By the way, let's increase it even more: https://travis-ci.org/status-im/status-go/builds/289424066#L530

Reply:
Okay, will do.